Core/src/Geometry/TrackingGeometryBuilder.cpp

```diff
@@ -52,8 +52,9 @@ Acts::TrackingGeometryBuilder::trackingGeometry(
     const IMaterialDecorator* materialDecorator =
         m_cfg.materialDecorator ? m_cfg.materialDecorator.get() : nullptr;
     // build and set the TrackingGeometry
-    trackingGeometry.reset(new TrackingGeometry(
-        highestVolume, materialDecorator, m_cfg.geometryIdentifierHook));
+    trackingGeometry.reset(
+        new TrackingGeometry(highestVolume, materialDecorator,
+                             m_cfg.geometryIdentifierHook, logger()));
```

Comment:
I think this should be the same:
```suggestion
                             m_cfg.geometryIdentifierHook, logger));
```

Reply:
To my understanding not realy. We need Logger& here. m_logger is unique ptr and we could call `*m_logger.get()` but having the logger() method returning already ref it seems better to just use it.